ReadMe.md

```diff
@@ -51,10 +53,31 @@ setTimeoutAt(function (msg) {
 }, Date.now() + 5000, 'Hello world');
 ```
 
+To cancel a `Timeout`, use the instance's `clear` method, or pass it as an argument to `clearTimeout`:
+
+```js
+const st = require('safe-timers');
+const setTimeout = st.setTimeout;
+const clearTimeout = st.clearTimeout;
+
+let timeout1 = setTimeout(function (msg) {
+  console.log(msg);
+}, 5000, 'Hello world');
+let timeout2 = setTimeout(function (msg) {
+  console.log(msg);
+}, 5000, 'Hello world');
+
+timeout1.clear(); // this clears the timeout
+clearTimeout(timeout2); // this also clears the timeout!
+```
+
+### Interval
+
+To create an `Interval`, use the `setinterval` function:
```

Comment:
The "i" in `setInterval` should be capitalized (uppercase).

Reply:
Thanks for catching that!